k8s/vpc-workflow.yaml

```diff
@@ -4,9 +4,9 @@ metadata:
   name: vpc-workflow
 spec:
   workflowName: aws_vpc_yaml
-  refreshTime: 65
+  refreshTime: 60
```

Comment:
The [spec](https://github.com/lyraproj/lyra/blob/master/k8s/lyra_v1alpha1_workflow_crd.yaml#L46) has a limit of '60':

Reply:
good catch - @markfuller and i ran into this too, i think it was an accidental checkin.